app/actions/ActionTypes.ts

```diff
@@ -408,3 +408,13 @@ export const Reaction = {
   ADD: generateStatuses('Reaction.ADD') as AAT,
   DELETE: generateStatuses('Reaction.DELETE') as AAT,
 };
+
+/**
+ *
+ */
```

Comment:
nice

Reply:
I agree that this is stupid, but it's the way it's done for all otheraction types as well...

Reply:
Just because something is bad doesn't mean the new stuff must be bad, or something, said by someone

Reply:
Kinda true, I might fix